chia/farmer/farmer.py

```diff
@@ -255,7 +259,7 @@ async def _pool_post_farmer(
                 ) as resp:
                     if resp.ok:
                         response: Dict = json.loads(await resp.text())
-                        self.log.info(f"POST /farmer response: {response}")
+                        self.log.info(f"POST /farmer response: {response}")  # todo which wallet / nft
```

Comment:
Whoops - that was a note to myself to do another PR.

What I mean is, it would be good of we say which pool or which plotnft in the log message, in case the user has several.